Docs 2210 (#1228)
MultiSafepay/docs: content/faq/getting-started/onboarding-new-merchants.md

    - /faq/getting-started/account-approval/
    - /faq/getting-started/onboarding
---
Onboarding refers to the steps you need to follow to create a live account with MultiSafepay. When new merchants [sign up](https://merchant.multisafepay.com/signup) and we create a new account, we first perform several [know your customer](https://en.wikipedia.org/wiki/Know_your_customer) checks on the:
* account holder.
* associated bank account.
* merchant.
* associated website, and the products and services offered to make sure they comply with our policies. See [Prohibited products and services](https://docs.multisafepay.com/faq/getting-started/prohibited-products-and-services/).

To enable you to process payments as quickly as possible, we divide the onboarding checks into two phases:

* **Identification**: Once we have established the identity of the account holder, you can start collecting customer payments.
* **Verification**: Once we have verified the identity of the account holder, you can receive payouts to your business bank account.

{{< br >}}
## Know your customer checks

### Documents and information

> You can upload all of the required documents and information in your MultiSafepay Control account → [Onboarding flow](https://merchant.multisafepay.com/onboarding):

Reviewer: ```suggestion
You can upload all of the required documents and information in your MultiSafepay Control > [Business details](https://merchant.multisafepay.com/onboarding):
```

Reviewer: Even though the URL suggests it's named onboarding, the heading of the page is 'Business details', so we should probably match that.
<img width="1171" alt="Screenshot 2021-04-15 at 16 17 25" src="https://user-images.githubusercontent.com/73658536/114884455-12f57280-9e06-11eb-83d9-d749531a5199.png">

* Company details including address, VAT number, and Chamber of Commerce number
* A link to the website you receive payments through, if used 
* Information about the [ultimate beneficial owners](https://docs.multisafepay.com/faq/getting-started/ultimate-beneficial-owner-form/) of your company, including color scans of valid identity documents. We recommend using the [KopieID app](https://www.rijksoverheid.nl/onderwerpen/identiteitsfraude/vraag-en-antwoord/veilige-kopie-identiteitsbewijs) to hide personal information on passports or IDs.
* A payment of 1 EUR from your business bank account. Alternatively, you can provide a copy or screenshot of a bank statement that includes your IBAN and company name.

The trade name of your business bank account must exactly match at least one of the trade name listed on your Chamber of Commerce extract.

**Note:** The documents and information mentioned above are the standard requirements for performing a check. We may ask you to provide additional documents, e.g. for an unusual company structure.

### Credit card payments
To process credit card payments, you must provide your:

> * minimum order value.

Reviewer: ```suggestion
* Minimum order value
```

> * average order value.

Reviewer: ```suggestion
* Average order value
```

> * maximum order value.

Reviewer: ```suggestion
* Maximum order value
```

> * annual turnover in credit card transactions.

Reviewer: ```suggestion
* Annual turnover in credit card transactions
```
